src/GafferUIModule/PathListingWidgetBinding.cpp

```diff
@@ -1583,6 +1581,7 @@ class PathModel : public QAbstractItemModel
 
 		void treeViewExpanded( const QModelIndex &index )
 		{
+			static_cast<Item *>( index.internalPointer() )->treeViewExpansionChanged( true );
 			if( m_modifyingTreeViewExpansion )
 			{
 				// When we're modifying the expansion ourselves, it's to mirror
```

Comment:
Might be worth adding a comment here explaining why, unlike m_expandedPaths, we do need to mirror to treeViewExpansionChanged?  I think it makes sense, but more based on your comments in the PR than what I'm seeing here.

Reply:
Seems reasonable to me.  Looks to good to merge.